content/en/tracing/profiling/_index.md

```diff
@@ -54,20 +54,26 @@ The Datadog Profiler requires [Java Flight Recorder][1]. The Datadog Profiling l
 
 - Because profiles are sent directly to Datadog without using the Datadog Agent, you must pass a valid [Datadog API key][4].
 
-- As an alternative to passing arguments, you can use environment variable to set those parameters:
+- For advanced setup of the profiler or to add tags like `service` or `version`, use environment variables to set the parameters:
 
-| Arguments                     | Environment variable      | Description                                       |
-| ----------------------------- | ------------------------- | ------------------------------------------------- |
-| `-Ddd.profiling.enabled`      | DD_PROFILING_ENABLED      | Set to `true` to enable profiling.                |
-| `-Ddd.profiling.api-key-file` | DD_PROFILING_API_KEY_FILE | File that should contain the API key as a string. |
-|                               | DD_PROFILING_API_KEY      | Datadog API key.                                  |
-| `-Ddd.site`                   | DD_SITE                   | Destination site for your profiles (versions 0.48+). Valid options are `datadoghq.com` for Datadog US site (default), and `datadoghq.eu` for the Datadog EU site. |
+| Environment variable                             | Type          | Description                                                                                      |
+| ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
+| `DD_PROFILING_ENABLED`                           | Boolean       | Alternate for `-Ddd.profiling.enabled` argument. Set to `true` to enable profiling.               |
+| `DD_API_KEY`                                     | String        | Alternate for `-Ddd.profiling.api-key-file` argument. The [Datadog API key][4] to use when uploading profiles. |
```

Comment:
IMO if we didn't support this back to the initial release, which I don't think we did, we should do what we did for python (and many projects do) and show the version when it was introduced, and keep the old setting but say it's deprecated. Otherwise someone who's running one of those early versions will follow this guide and fail.

Reply:
I see! We should track those on this sheet: https://docs.google.com/spreadsheets/d/1qrz2msAPWcCD8X6zkaEHZ9ut80Uuu3x1aNexV33_cG0/edit#gid=0 

For now, would you have state on what version this is supported from? Is it 0.48 +?